Implement run command.

mareksuscak opened the pull request:
This PR proposes a new command that runs the service but doesn’t start it at login (nor boot).

It is based on the discussion in PR #87. 

Feedback is welcome.

MikeMcQuaid commented on cmd/brew-services.rb:
```diff
@@ -213,11 +221,29 @@ def cleanup
+        
         stop(service) if service.loaded?
-        start(service)
+        start(service) if was_installed
```
Make this an `if/else`

MikeMcQuaid commented on cmd/brew-services.rb:
```diff
@@ -213,11 +221,29 @@ def cleanup
+    
+    # Run a service.
+    def run(target)
+      if target.is_a?(Service)
```
`if target.is_a?(Service) && target.loaded?`

MikeMcQuaid commented on cmd/brew-services.rb:
```diff
@@ -213,11 +221,29 @@ def cleanup
+      
+      Array(target).each do |service|      
+        safe_system launchctl, "load", "-w", target.plist
+        $?.to_i.nonzero? ? odie("Failed to start `#{service.name}`") : ohai("Successfully started `#{service.name}` (label: #{service.label})")
```
Make this an `if/else`

MikeMcQuaid commented:
Looking good so far! A few nits. May also want to add this as a test to `.travis.yml` too.

mareksuscak commented:
Done. As you can tell, I'm not a Ruby guy. Anyway, removed all the nits and added a test. Maybe I should add one more to test if the `*.plist` was not installed? Also I did not squash the commits to make it easier to review. Let me know if you want me to do so afterwards.

MikeMcQuaid commented:
Great work! Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!
